pkg/connector/helpers.go

```diff
@@ -3,23 +3,17 @@ package connector
 import (
 	"fmt"
 	"strconv"
-	"strings"
 
 	v2 "github.com/conductorone/baton-sdk/pb/c1/connector/v2"
 	"github.com/conductorone/baton-sdk/pkg/pagination"
 )
 
 func formatObjectID(resourceTypeID string, id int64) string {
-	return fmt.Sprintf("%s:%d", resourceTypeID, id)
+	return fmt.Sprintf("%c%d", resourceTypeID[0], id)
```

Comment:
`%c` will print the character "represented by the corresponding Unicode code point", are you sure this is what you want?

Reply:
Yes